packages/contracts-communication/contracts/InterchainClientV1.sol

```diff
@@ -115,6 +115,8 @@ contract InterchainClientV1 is Ownable, InterchainClientV1Events, IInterchainCli
         icTx.dbNonce = IInterchainDB(interchainDB).writeEntryWithVerification{value: verificationFee}(
             icTx.dstChainId, icTx.transactionId, srcModules
         );
+        IExecutionFees(executionFees).addExecutionFee{value: executionFee}(icTx.dstChainId, transactionId);
```

Comment:
This will revert if execution fee is zero (based on current `ExecutionFees`). I think calling here conditionally makes more sense than no-op in ExecutionFees with zero value passed.

Reply:
This was fixed in c2271d906833d60643fa1f281665e0e3ee0f2f63